apps/site/src/theme/components/navigation/mui-menu-item-theme-options.ts

```diff
@@ -0,0 +1,129 @@
+import {
```

Comment:
Are these taken from HASH?

Reply:
Yes

Reply:
Just wondering if we should abstract this theming to a package or include it in the design-system to be imported as we need it. Had to do the same thing for the address block and it feels weird copying a whole file just to be able to use it once somewhere that isn't the HASH app.

Reply:
Out of scope for this PR ofc.

Reply:
It would be good to get this into the design system package. cc @kachkaev (as we had discussed adding more components to it), also @CiaranMn for visibility.